cypress/fixtures/ChartDataBoys.json

```diff
@@ -0,0 +1,3280 @@
+{
+    "hcfa_b": {
+        "categoryMetadata": {
+            "label": "Head circumference for age",
+            "gender": "Boy"
+        },
+        "datasets": {
+            "0 to 13 weeks": {
+                "datasetValues": [
+                    {
+                        "SD0": 34.5,
+                        "SD1": 35.7,
+                        "SD2": 37,
+                        "SD3": 38.3,
+                        "SD1neg": 33.2,
```

Comment:
If you want, you can extract the datasetValues to an own fixture, do it in both files if you do. If the import is difficult then dont